tests/shell/shell_recorder/README.md

```diff
@@ -0,0 +1,331 @@
+# Shell Recorder #
+
+## Configuration ##
+
+### Mountpoint: ###
+
+This is the only configuration Variable that has to be set. It is used to determine where the shell_recorder should look for changes.
+e.g. `Mountoint: user/test` tells the shell_recorder that you will be working under `user/test`.
+
+### REPLAY! ###
+
+When `REPLAY!` is set the shell_recorder runs the input script twice and compars the results.
```

Comment:
I think this should work differently: The idea is that you record once the output of an input script. Then we add this recorded file to libelektra git repo. From then on, the build server replays the same input script and sees if the output still matches.